### ISSUE TITLE
badge: <pt_BR> for <applesafe>

### ISSUE THREAD

**Issue by carmoswitch** [revoltchat/translations#315]
### What language(s) did you translate?

Português BR

### What is your Revolt ID?

01HZPNFSZ07WPF4WDM8TD4F03S

### Link to Weblate profile

https://translate.revolt.chat/user/mateus/

### Validations

- [X] The language has a maintainer (see the full list [here](https://github.com/revoltchat/translations#languages)).
- [X] You agree that your translations are to a reasonable standard.

**Comment by Rexogamer** [comment]
cc @yanndere

**Comment by applesafe** [comment]
Would this be to wait for his approval?

**Comment by applesafe** [comment]
> cc @yanndere

I believe he is missing 🤷🏽

**Comment by bobspook** [comment]
> If this request is still active and pending action, please let us know by replying to this comment. Thanks!

yes it is still pending, I am waiting for approval (but maintainer is inactive)

**Comment by matoeus** [comment]
> If this request is still active and pending action, please let us know by replying to this comment. Thanks!

Oh, As I changed my nickname I ended up forgetting to renew the link. 

https://translate.revolt.chat/user/mateus/

**Comment by DeclanChidlow** [comment]
Badge applied. Will reflect shortly.
